src/gui/Src/Gui/LogView.cpp

```diff
@@ -415,6 +420,27 @@ void LogView::copyToDebuggeeNotes()
     emit Bridge::getBridge()->setDebuggeeNotes(Notes);
 }
 
+void LogView::searchInLogSlot() {
+    dialogSearchInLog->show();
+
+    if (dialogSearchInLog->exec() == QDialog::Accepted) {
+        QList<QTextEdit::ExtraSelection> extraSelections;
+        QColor highlight = QColor(Qt::blue);

```

Comment:
Here it would be good to reuse the existing ConfigColor for highlighting and match the highlighting behavior in the disassembler (change text color instead of background).